models/DataObject/ClassDefinition/Data/Date.php

```diff
@@ -487,4 +487,18 @@ protected function doGetDefaultValue($object, $context = [])
 
         return null;
     }
+
+    /**
+     * @param \DateTimeInterface|null $oldValue
+     * @param \DateTimeInterface|null $newValue
+     *
+     * @return bool
+     */
+    public function isEqual($oldValue, $newValue)
+    {
+        $oldValue = $oldValue instanceof \DateTimeInterface ? $oldValue->format('Y-m-d') : null;
+        $newValue = $newValue instanceof \DateTimeInterface ? $newValue->format('Y-m-d') : null;
```

Comment:
can we compare the timestamps instead ?

Reply:
time should be ignored for comparison...right?